feed/feed.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/dist/css/styles.css">
+    <link rel="stylesheet" href="/src/scss/styles.scss">
```

Comment:
Remove this because it is the source file and SASS is already helping to compile it to css .